### PR TITLE
Adds a new plant for hydroponics - ambrosia cruciatus.

### PR DESCRIPTION
**Pull request by ghost**
It has bicardine, nutriment, bath salts, tirizine and kelotane. Also, while growing in a hydroponic tray it looks exactly as an ambrosia gaia, so it wouldn't look suspicious for observators. 
It is only obtainable by buying it's seeds from traitor cook or botanist uplink for 2 TC.

# What is the purpose of it?

I think that having a traitor-unique plant would be interesting idea, that can, maybe add more strategies for traitor botanists.

# Wiki Documentation

Add the description of this plant to "Guide to plants"
Add the description of "Ambrosia Cruciatus seeds" to "Syndicate Items"

# Changelog

Adds a new plant/traitor item for botany.

:cl:  
rscadd: Added new thing
/:cl:

### CODE REVIEW

**Comment by bruhlookatthisdood** [comment]
maybe instead of looking EXACTLY like it it could be a bit darker than the original?

**Comment by bruhlookatthisdood** [comment]
maybe instead of looking EXACTLY like it it could be a bit darker than the original?

**Comment by ghost** [comment]
> maybe instead of looking EXACTLY like it it could be a bit darker than the original?

Why? Kinda, it can be just rightclicked to see that it is not normal ambrosia gaia, so making it look different from normal ambrosia doesn't make sense.

**Comment by bruhlookatthisdood** [comment]
nvm then

**Comment by UselessTheremin** [comment]
why is it also in traitor cook uplink tho

**Comment by Chubbygummibear** [comment]
Stealth plant is a pretty cool idea that is only really used in the poisoned apple, so i think this could be a pretty neat thing, especially since it's disguised as gaia which is often the end all be all. However i do worry that having kelotane and bicardine in it is a bit strong since you could rip those into a different healing plant, alongside libital and aiuri. Also the stamina damage from the tirizine is kinda negated by the stamina regen from bath salts.

and i think you might want to change the datum lmao
"/datum/uplink_item/role_restricted/explosive_hot_potato"

Oh and I do think this is kinda useless for a chef because just getting the seed still means you have to grow it, and for the best results you basically have to do the full botany thing... as chef

**Comment by ilikesaying** [comment]
the chef could grind the ambrosia down and separate the reagents for usage as Condiments, a sandwich laced with tirizine for the target that makes em fall asleep, a donk-pocket laced with bath salts, bicardine and kelotane for combat

**Comment by ToasterBiome** [comment]
> why is it also in traitor cook uplink tho

Cook gets botany access on lowpop

**Comment by Chubbygummibear** [comment]
> > why is it also in traitor cook uplink tho
> 
> Cook gets botany access on lowpop

i worry this would get meta'd in that if you see the chef with "ambrosia gaia" during lowpop everyone is going to be suspicious of them and in addition to the fact that they could have "gaia" yet none of the trays are glowing to indicate earthsblood

**Comment by ToasterBiome** [comment]
> > > why is it also in traitor cook uplink tho
> > 
> > 
> > Cook gets botany access on lowpop
> 
> i worry this would get meta'd in that if you see the chef with "ambrosia gaia" during lowpop everyone is going to be suspicious of them and in addition to the fact that they could have "gaia" yet none of the trays are glowing to indicate earthsblood

admin issue

**Comment by ghost** [comment]
> Also the stamina damage from the tirizine is kinda negated by the stamina regen from bath salts.

You can just separate it’s ingridient’s in different plants, so no problem.

File: code/modules/hydroponics/grown/ambrosia.dm [hunk 1]
```diff
@@ -77,3 +77,22 @@
 	seed = /obj/item/seeds/ambrosia/gaia
 	wine_power = 70
 	wine_flavor = "the earthmother's blessing"
+
+
+
+// Ambrosia Cruciatus
+/obj/item/seeds/ambrosia/cruciatus
+    name = "pack of ambrosia cruciatus seeds"
+    desc = "A special ambrosia type, mutated at syndicate laboratories."
```
Comment (nesisknesis) [inline code/modules/hydroponics/grown/ambrosia.dm]:
```suggestion
    desc = "A special ambrosia type, mutated in syndicate laboratories."
```

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -2230,3 +2230,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/stamp/syndiround
 	cost = 1
 	illegal_tech = FALSE
+
+
+
+/datum/uplink_item/role_restricted/ambrosia_cruciatus_seeds
+    name = "Ambrosia Cruciatus Seeds"
+    desc = "A pack of ambrosia seeds, that contain usefull chemicals inside. This chemicals could be used for healing yourself, or killing other people."
```
Comment (nesisknesis) [inline code/modules/uplink/uplink_items.dm]:
```suggestion
    desc = "A pack of ambrosia seeds that contain useful chemicals inside. These chemicals can be used for healing yourself, or killing other people."
```

**Review by nesisknesis — COMMENTED** [review]
Does the ambrosia gaia sprite need to be duplicated so that it appears for ambrosia cruciatus?

This bit isn't apart of my review but I think this would work better as an overtly illegal plant. People unfortunately will likely just inspect gaia as they're passing by and see that its phony which has metagaming issues. If its something actaully illegal like super space cannabis or something and looks illegal it'd mean the botanist has to grow it in the back of maintenance so you don't have to faff about with metagaming and forces them to be smart.

**Comment by ghost** [comment]
> Does the ambrosia gaia sprite need to be duplicated so that it appears for ambrosia cruciatus?

Maybe yes, but i am not sure, since i am very new into modding ss13. When i didnt duplicate ambrosia gaia sprites it simply was invisible.

> This bit isn't apart of my review but I think this would work better as an overtly illegal plant. People unfortunately will likely just inspect gaia as they're passing by and see that its phony which has metagaming issues. If its something actaully illegal like super space cannabis or something and looks illegal it'd mean the botanist has to grow it in the back of maintenance so you don't have to faff about with metagaming and forces them to be smart.

I think metagaming would be more an "admin issue" as said one of commentators, and also yes, botanist can just grow it in maintenance like you said. Or just simply use the seed pack from the uplink just to extract genes from it, without planting it in a tray.

**Review by nesisknesis — APPROVED** [review]

**Comment by Chubbygummibear** [comment]
> Maybe yes, but i am not sure, since i am very new into modding ss13. When i didnt duplicate ambrosia gaia sprites it simply was invisible.

i copied your code into my test server and i believe the reason it wasn't showing up was because you set the species to ambrosia_cruciatus. the species tells the plant what icons to use when growing, so changing the species to 
species = "ambrosia_gaia"
makes the cruciatus show as gaia without the extra sprites added.

Additionally, and i'm not sure if this was on purpose, so sorry if that's what you intended, but when you harvest cruciatus, you get the default green ambrosia vulgaris branches instead of the gold leaves of gaia. you can copy the guts of the ambrosia gaia reagent container and paste it under 
/obj/item/reagent_containers/food/snacks/grown/ambrosia/cruciatus
to get a fake gold leaf, just remember to change the name and whatnot to what you want

**Comment by ghost** [comment]
> Additionally, and i'm not sure if this was on purpose, so sorry if that's what you intended, but when you harvest cruciatus, you get the default green ambrosia vulgaris branches instead of the gold leaves of gaia. you can copy the guts of the ambrosia gaia reagent container and paste it under /obj/item/reagent_containers/food/snacks/grown/ambrosia/cruciatus to get a fake gold leaf, just remember to change the name and whatnot to what you want

It was on purpose.

**Comment by ghost** [comment]
Can i have this merged?

**Comment by ghost** [comment]
Fuck, because of this shitty traitor plant i can't make a PR with more funny feature.

File: code/modules/uplink/uplink_items.dm [hunk 3]
```diff
@@ -2230,3 +2230,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/stamp/syndiround
 	cost = 1
 	illegal_tech = FALSE
+
+
```
Comment (JamieD1) [inline code/modules/uplink/uplink_items.dm]:
```suggestion
```

**Comment by Cartlord** [comment]
I don't like this, because I think it'll cause confusion - I already know of at least one other server that has "ambrosia cruciatis", and on that server it's also a Traitor-only seed, but it looks like Vulgaris (since they don't have Gaia there) and it's laced with a special toxin that's so weird and fucked-up that it's considered a method of removing people from the round entirely.

File: code/modules/hydroponics/grown/ambrosia.dm [hunk 4]
```diff
@@ -77,3 +77,22 @@
 	seed = /obj/item/seeds/ambrosia/gaia
 	wine_power = 70
 	wine_flavor = "the earthmother's blessing"
+
+
+
+// Ambrosia Cruciatus
```
Comment (JamieD1) [inline code/modules/hydroponics/grown/ambrosia.dm]:
```suggestion

// Ambrosia Cruciatus
```

**Comment by ghost** [comment]
> I don't like this, because I think it'll cause confusion - I already know of at least one other server that has "ambrosia cruciatis", and on that server it's also a Traitor-only seed, but it looks like Vulgaris (since they don't have Gaia there) and it's laced with a special toxin that's so weird and fucked-up that it's considered a method of removing people from the round entirely.

My ambrosia cruciatus is definitely not so OP.

**Comment by Chubbygummibear** [comment]
i think your cruciatus sprites are still in the growing.dmi after switching to the gaia sprites

**Comment by JamieD1** [comment]
I like the hidden plant idea, the idea of syndicate chems in the plant can be cool, but need to be a bit more inspired, and disguising it as gaia is too easy to discern to players who have been around for a while.

**Pull request closed without merging**